update_gvf/update_GVF.f90

```diff
@@ -39,7 +39,7 @@ PROGRAM update_GVF
   real, allocatable :: vegfrc_max(:,:)
   real, allocatable :: vegfrc_min(:,:)
   real, allocatable :: field2d(:,:)
-  real(8), allocatable,target :: field2d8b(:,:)
+  real, allocatable,target :: field2d4br(:,:)
```

Comment:
why we don't need to use "real(r_single)" here?

Reply:
No reason.